systems/Query/diagrams/07-wikidata-deployment-single-host.drawio.svg

```diff
@@ -0,0 +1,313 @@
+                            <div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; ">
+                                Data Loader
+                            </div>
+                        </div>
+                    </div>
+                </foreignObject>
+                <text x="117" y="274" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">
+                    Data Loader
+                </text>
+            </switch>
+        </g>
+        <path d="M 117 240 L 117 220 L 117 225 L 117 211.37" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="stroke"/>
+        <path d="M 117 206.12 L 120.5 213.12 L 117 211.37 L 113.5 213.12 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/>
+        <g transform="translate(-0.5 -0.5)">
+            <switch>
+                <foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility">
+                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 220px; margin-left: 137px;">
+                        <div style="box-sizing: border-box; font-size: 0; text-align: center; ">
+                            <div style="display: inline-block; font-size: 11px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; background-color: #ffffff; white-space: nowrap; ">
+                                http
+                            </div>
+                        </div>
+                    </div>
+                </foreignObject>
+                <text x="137" y="224" fill="#000000" font-family="Helvetica" font-size="11px" text-anchor="middle">
+                    http
+                </text>
+            </switch>
+        </g>
+        <rect x="57" y="145" width="120" height="60" fill="url(#mx-gradient-d5e8d4-1-e1d5e7-1-s-0)" stroke="#82b366" pointer-events="all"/>
+        <g transform="translate(-0.5 -0.5)">
+            <switch>
+                <foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility">
+                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 175px; margin-left: 58px;">
+                        <div style="box-sizing: border-box; font-size: 0; text-align: center; ">
+                            <div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; ">
+                                wdqs
```

Comment:
same here, maybe explain this somewhere or drop it.

Reply:
Done!